romana/cmd/host.go

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2016 Pani Networks
+// All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/romana/core/common"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// hostCmd represents the host commands
+var hostCmd = &cobra.Command{
+	Use:   "host [add|show|list|remove]",
+	Short: "Add, Remove or Show hosts for romana services.",
+	Long: `Add, Remove or Show hosts for romana services.
+
+host requires a subcommand, e.g. ` + "`romana host add`." + `
+
+Romana is a new Software Defined Network solution specifically
```

Comment:
Do we need this text here as part of every commands info/help page? This could get repetirive quickly, no? I think if we would have online docs (with examples and such) for each command then it would be helpful to include that specific link to that specific doc in the info text. But the general blurb about Romana is probably not too helpful. It's good to have a short and long version, though. In absence of the link to command-specific online docs we could present command specific examples or notes here in the long text.